### DISCUSSION TITLE
Change default color palette

### DISCUSSION THREAD

**Discussion by SnowyTheDragon** [schismtracker/schismtracker#440] [Q&A]
Is there any way to change the default palette? I don't really like the Camouflage one and don't want to have to do it manually every time I open the program.

**Comment by alexmyczko** [comment]
ESC (Main Menu), Settings Menu, Palette Editor, Predefined Palettes... ah and it never saves it anywhere and `schismtracker -h` also doesn't allow to set a different one...

**Comment by jangler** [comment]
I remember there being a way to do this, but I don't have Schism installed right now...

**Comment by alexmyczko** [comment]
One would need to check the source for this kind of stuff: https://github.com/schismtracker/schismtracker/blob/master/docs/configuration.md

Ah found it with in `scripts/itcfg.py`, `print("palette_cur=%s" % palette)`

Like `palette_cur=VC_ZVBjrV1WiZS9am_lxlq9bWdixk9Q4GVC_ZV7dbcV1WiZS` gives you different colors, probably not what you want. Ahhhh look at `scripts/palette.py`

You want to reverse that...
```
$ python3 ./palette.py VC_ZVBjrV1WiZS9am_lxlq9bWdixk9Q4GVC_ZV7dbcV1WiZS
/*  0 */ {32, 13, 37},
/*  1 */ {36, 32, 12},
/*  2 */ {47, 55, 32},
/*  3 */ { 2, 33, 46},
/*  4 */ {36, 29, 10},
/*  5 */ {38, 50, 37},
/*  6 */ {49, 61, 49},
/*  7 */ {54, 10, 39},
/*  8 */ {33, 41, 46},
/*  9 */ {61, 48, 10},
/* 10 */ {27,  5, 17},
/* 11 */ {32, 13, 37},
/* 12 */ {36, 32,  8},
/* 13 */ {41, 39, 40},
/* 14 */ {32,  2, 33},
/* 15 */ {46, 36, 29},
```

**Comment by mrpapersonic** [comment]
You have to go to a different page to save the palette configuration, I believe it's Preferences and you have to click Save all Preferences.

**Comment by BKVad1m** [comment]
> You have to go to a different page to save the palette configuration, I believe it's Preferences and you have to click Save all Preferences.

Also when modifying palette can saved automatically

**Comment by BKVad1m** [comment]
in #245 and #294
